cmd/cluster-autoscaler/cluster_autoscaler_test.go

```diff
@@ -35,7 +36,7 @@ func TestClusterAutoscalerScaleUpFromNonZero(t *testing.T) {
 		testkit.RetainResourcesOnFailure(),
 	)
 
-	kc := h.KubernetesCluster(t)
+	kc := h.KubernetesCluster(t, ktesting.WithRandomClusterID())
```

Comment:
By default, KubernetesCluster looks for clusters whose names start with testkit-. If one is found, it returns a cluster struct containing that cluster’s kubeconfig information. With this change, it is guaranteed that a new cluster will always be created.